routes/project/location/postgres.rb

```diff
@@ -84,16 +87,15 @@ class Clover
           request_body_params = Validation.validate_request_body(json_params, required_parameters)
           parsed_cidr = Validation.validate_cidr(request_body_params["cidr"])
 
-          DB.transaction do
+          firewall_rule = DB.transaction do
             PostgresFirewallRule.create_with_id(
               postgres_resource_id: pg.id,
               cidr: parsed_cidr.to_s
-            )
-            pg.incr_update_firewall_rules
+            ).tap { pg.incr_update_firewall_rules }
           end
 
           if api?
-            Serializers::Postgres.serialize(pg, {detailed: true})
+            Serializers::PostgresFirewallRule.serialize(firewall_rule, {detailed: true})
```

Comment:
Probably worth adding a test for this change.

Reply:
I think it does get tested, albeit somewhat indirectly, due to the committee tests. ie there are tests that call this and if it didn't return what the spec asks for it would error. That said, it may be worth it to also be more explicit.

Reply:
Yes. Committee is quite strict and uniform. When doing the prototype, the cost was measurable but virtual. We can rebate some of that via fewer tests for form.

Reply:
Likewise, I assume the committee tests for structure, whereas we would want a test for expected values.

Reply:
I added one puny test for a value [not that the firewall rules for postgres have many values to set] as well, that should probably do the job with committee so far